local-cli/server/runServer.js

```diff
@@ -190,7 +191,8 @@ function getPackagerServer(args, config, reporter) {
     resetCache: args.resetCache,
     sourceExts: args.sourceExts.concat(defaultSourceExts),
     transformModulePath: transformModulePath,
-    transformCache: TransformCaching.useTempDir(),
+    transformCache: args.cacheDir || TransformCaching.useTempDir(),
+    useDeltaBundler: false,
```

Comment:
@hramos I didn't see this before, but I suspect this might break some of the Metro team's plans. Will it break the import if I edit again?

Reply:
Go for it - the PR did not land, anyway.